README.md

```diff
@@ -1,6 +1,7 @@
 Authors Haven - A Social platform for the creative at heart.
-=======
-[![Build Status](https://travis-ci.com/andela/ah-bird-box.svg?branch=develop)](https://travis-ci.com/andela/ah-bird-box)
+
+[![Build Status](https://travis-ci.com/andela/ah-bird-box.svg?branch=develop)](https://travis-ci.com/andela/ah-bird-box) [![Coverage Status](https://coveralls.io/repos/github/andela/ah-bird-box/badge.svg?branch=ch-integrate-coveralls-163388245)](https://coveralls.io/github/andela/ah-bird-box?branch=ch-integrate-coveralls-163388245)
```

Comment:
I suggest you ensure the coveralls badge point to develop branch.

Reply:
Done. It should be okay now.